### ISSUE TITLE
Alt Text Field not deactivated in Media Editor

### ISSUE THREAD

**Issue by NolanHill** [coyote-team/coyote-wp#85]
Kathleen just noticed that the media editor page does not have the deactivated alt text field/link to coyote
e.g. https://chsdmdev.wpengine.com/wp-admin/post.php?post=39462&action=edit

**Comment by sinabahram** [comment]
This is without Gutenberg, correct?

**Comment by NolanHill** [comment]
Yes

**Comment by sinabahram** [comment]
Pinging @jkva  on this

**Comment by jkva** [comment]
Thanks for the find! Fixed in v1.6.1.
